core/kt-osrd-utils/src/main/kotlin/fr/sncf/osrd/utils/units/Duration.kt

```diff
@@ -0,0 +1,31 @@
+package fr.sncf.osrd.utils.units
+
+import kotlin.math.absoluteValue
+
+@JvmInline
+value class Duration(val milliseconds: Long) : Comparable<Duration> {
```

Comment:
Can you add a toString method, like the one Distance has? It helps display the type while debugging